cli/cmd/version.go

```diff
@@ -34,45 +34,110 @@ const versionCheckInfo = `Daily version check is %s.
 Keptn will%s collect statistical data and will%s notify about new versions and security patches for Keptn. Details can be found at: https://keptn.sh/docs/0.7.x/reference/version_check
 ---------------------------------------------------
 `
-const enableVersionCheckMsg = `* To %s the daily version check, please execute:
+const setVersionCheckMsg = `* To %s the daily version check, please execute:
  - keptn set config AutomaticVersionCheck %s
 
 `
 const keptnReleaseDocsURL = "0.7.x"
 
+const updateInfoMsg = `
+Please visit https://keptn.sh for more information about updating.
+`
+
+const disableVersionCheckMsg = "To disable this notice, run: '%s set config AutomaticVersionCheck false'"
+
 // KubeServerVersionConstraints the Kubernetes Cluster version's constraints is passed by ldflags
 var KubeServerVersionConstraints string
 
 // versionCmd represents the version command
 var versionCmd = &cobra.Command{
 	Use:     "version",
-	Short:   "Shows the CLI version of Keptn",
-	Long:    `Shows the CLI version of Keptn and a note when a new version is available.`,
+	Short:   "Shows the version of Keptn and Keptn CLI",
+	Long:    `Shows the version of Keptn and Keptn CLI, and a note when a new version is available.`,
 	Example: `keptn version`,
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("\nKeptn CLI version: " + Version + "\n")
-
-		configMng := config.NewCLIConfigManager()
-		cliConfig, err := configMng.LoadCLIConfig()
+		isLastCheckStale, err := isLastCheckStale()
 		if err != nil {
 			logging.PrintLog(err.Error(), logging.InfoLevel)
 			return
 		}
-		checkTime := time.Now()
-		if cliConfig.LastVersionCheck == nil || checkTime.Sub(*cliConfig.LastVersionCheck) >= time.Second {
+
+		var cliMsgPrinted, cliChecked, keptnMsgPrinted, keptnChecked bool
+
+		// Keptn CLI
+		fmt.Println("\nKeptn CLI version: " + Version)
+		if isLastCheckStale {
 			vChecker := version.NewVersionChecker()
-			vChecker.CheckCLIVersion(Version, false)
+			cliMsgPrinted, cliChecked = vChecker.CheckCLIVersion(Version, false)
 		}
-		if cliConfig.AutomaticVersionCheck {
-			fmt.Printf(versionCheckInfo, "enabled", "", "")
-			fmt.Printf(enableVersionCheckMsg, "disable", "false")
-		} else {
-			fmt.Printf(versionCheckInfo, "disabled", " not", " not")
-			fmt.Printf(enableVersionCheckMsg, "enable", "true")
+
+		// Keptn
+		keptnVersion, err := getInstalledKeptnVersion()
+		if err != nil {
+			logging.PrintLog(err.Error(), logging.InfoLevel)
+			return
+		}
+		fmt.Println("\nKeptn cluster version: " + keptnVersion)
```

Comment:
Please be careful with the return statement and the if-else block here.

If fetching the Keptn Cluster version info failed (e.g., because the user does not have access to the K8s cluster), we can still continue the remaining code (except for printing the Keptn cluster version, obviously).



Right now the output would be like this:
```
Keptn CLI version: develop
Failed to check if Keptn release is available in namespace keptn: ${ERROR_MESSAGE}

```

But in my opinion this would be okay:
```
Keptn CLI version: develop
Keptn cluster version:  Failed to check if Keptn release is available in namespace keptn: ${ERROR_MESSAGE}


Daily version check is enabled. 
Keptn will collect statistical data and will notify about new versions and security patches for Keptn. Details can be found at: https://keptn.sh/docs/0.7.x/reference/version_check
---------------------------------------------------
* To disable the daily version check, please execute:
 - keptn set config AutomaticVersionCheck false

```